library/Translate/src/Juvix/Contextify/Passes.hs

```diff
@@ -105,3 +122,45 @@ convertShunt :: Shunt.Application Sexp.T Sexp.T -> Sexp.T
 convertShunt (Shunt.Single e) = e
 convertShunt (Shunt.App s app1 app2) =
   Sexp.list [s, convertShunt app1, convertShunt app2]
+
+--------------------------------------------------------------------------------
+-- Record Transform
+--------------------------------------------------------------------------------
+
+-- Update this…
+
+-- Currently we aren't running a free variable algorithm on this to
+-- determine dependency order. This should be run to ensure that we are
+-- getting the order right. Also we need to pick a canonical ordering
+
+-- | @recordPi@ - Transforms record declarations into @Pi@ Type
+-- expressions
+-- - BNF Input Form
+--   1. (:record-d arg-1 type-1 … arg-n type-n)
+-- - BNF Output Form
+--   1. (:sigma (arg-1 :usage-hole type-1)
+--         (:sigma (arg-2 :usage-hole type-2)
+--            … (:sigma (arg-n :usage-hole type-n)
+--                  :unit)))
+recordPi ::
```

Comment:
Shouldn't this be recordSigma (or recordPiSigma)?
![Screenshot 2021-08-02 at 00 18 34](https://user-images.githubusercontent.com/2612856/127788249-b40b1ece-6e51-4e66-8b51-8d4f74492b7c.png)

Reply:
Yes I should call it recordSigma instead of `recordPi` good find.

The `Pi` part shouldn't be done here, as there is a generic encoding for the arguments that holds true regardless if it's a record or not